partcad/src/partcad/part_config.py

```
@@ -49,9 +50,12 @@
                     }
                 # All params are float unless another type is explicitly speciifed
-                elif (
-                    isinstance(param_value, dict) and not "type" in param_value
-                ):
+                elif isinstance(param_value, dict) and not "type" in param_value:
                     param_value["type"] = "float"
 
         return ShapeConfiguration.normalize(name, config)
+
+    @staticmethod
+    def get_manufacturing_data(part) -> PartConfigManufacturing:
```

Ruff (0.8.2)

Missing type annotation for function argument `part`

(ANN001)